### DISCUSSION TITLE
12 Hour RTC?

### DISCUSSION THREAD

**Discussion by DaTwo** [EdgeTX/edgetx#3190] [Q&A]
Is there a way to display the time in 12 hour format vs 24 hour format on the main screen? If not is there any chance to add a tick box in system settings to make it display this way?

**Comment by pfeerick** [comment]
Not that I am aware of. I presume it was done this way to minimise the amount of screen space needed for the clock, as at minimum one extra character would need to be shown for the meridiem indicator... Unless you are suggesting show the time without that?

**Reply by DaTwo** [reply]
With or without the meridiem indicator would be fine. Indicator would be best but if it wont fit then without is fine by me. As someone who rarely uses the 24 hour format having the option of the 12 hour format would be a welcome addition.

**Reply by DaTwo** [reply]
In looking at the screen. Comparing the date with the time it looks like 1 more character would fit perfectly.

**Comment by DaTwo** [comment]
@pfeerick ,is there any chance of making this happen?
